하미향/1st_session-counter/src/Counter/CounterPage.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import "../App.css";
+
+function CounterPage() {
+  const [count, setCount] = useState(0);
+
+  return (
+    <>
+      <div className="content">
+        <div className="count">
+          <h1>count : {count}</h1>
+        </div>
+        <div className="button">
+          <button
+            className="btn"
+            onClick={() => {
+              setCount((prev) => prev + 1);
+            }}
```

Comment:
직접 넣는것도 좋은 방법이지만, 함수로 따로 만들어서 사용해보세요!
직접 넣으면 가독성도 떨어질 뿐만 아니라, 오류가 생겼을 때 디버깅에도 어려움을 주니 UI와 기능은 분리해서 다루시는 방향으로 공부해보세요!

```suggestion
const increase = () => {
  setCount((prev) => prev + 1);
}
// ...
            onClick={increase}
```